packages/demo/src/smartFilterLoader.ts

```diff
@@ -28,6 +29,8 @@ export class SmartFilterLoader {
     private readonly _deserializer: SmartFilterDeserializer;
     private readonly _renderer: SmartFilterRenderer;
 
+    public readonly snippetUrl = "https://snippet.babylonjs.com";
+    public readonly onSmartFilterLoadedObservable: Observable<SmartFilter>;
     public readonly manifests: SmartFilterManifest[];
     public currentOptimizedSmartFilter: SmartFilter | undefined;
     public currentSmartFilter: SmartFilter | undefined;
```

Comment:
We don't use these two properties right? If so, I think we should remove them, and call this class stateless (doesn't track what was previously loaded) in the docstring.